queries.yaml

```diff
@@ -273,3 +273,28 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 9
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 14
+      properties: [is_bridge, is_tunnel, oneway]
+      where: >-
+        (kind == 'path' and zoom < 15) or
+        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
+    params:
+      source_layer: roads
+      start_zoom: 0
+      end_zoom: 16
+      properties: [name, ref, network]
+      where: >-
+        (kind == 'path' and zoom < 17) or
+        (kind in ['minor_road', 'rail'] and zoom < 15) or
+        (kind == 'major_road' and zoom < 12) or
+        (kind == 'highway' and zoom < 7)
```

Comment:
Or we might want to have a heavier weight tile set (like NEAT tiles but
consumable in simpler vector tile formats) eventually.

Agree it's purpose is to explore the tiles vs. all of OSM (since they
aren't that already), I think the important distinction may be a case where
more data is useful for *interactivity* (display on hover, click, etc.) vs.
just considering all initially visible labels. I think that dynamic
cartography is worth taking into account, though definitely don't want to
tilt so far that we lose the merging benefits for dominant use cases.


On Thu, Jan 7, 2016 at 11:20 AM, Nathaniel V. KELSO <
notifications@github.com> wrote:

> In queries.yaml
> <https://github.com/mapzen/vector-datasource/pull/468#discussion_r49091132>
> :
>
> > +      end_zoom: 14
> > +      properties: [is_bridge, is_tunnel, oneway]
> > +      where: >-
> > +        (kind == 'path' and zoom < 15) or
> > +        (kind in ['minor_road', 'major_road', 'highway', 'rail'] and zoom < 13)
> > +  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
> > +    params:
> > +      source_layer: roads
> > +      start_zoom: 0
> > +      end_zoom: 16
> > +      properties: [name, ref, network]
> > +      where: >-
> > +        (kind == 'path' and zoom < 17) or
> > +        (kind in ['minor_road', 'rail'] and zoom < 15) or
> > +        (kind == 'major_road' and zoom < 12) or
> > +        (kind == 'highway' and zoom < 7)
>
> Explorer should explore what's in tiles. If it wants raw OSM, it should
> connect directly with overpass-turbo or similar APIs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mapzen/vector-datasource/pull/468/files#r49091132>.
>